docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -146,8 +148,10 @@ class [[eosio::contract]] smrtcontract : public contract {
 1. Use the `eosio::kv::table::index` template class.
 2. Pass as the first parameter the name of the index. It must be a qualified `eosio:name`, see documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
```

Comment:
To match the change to second parameter maybe:
"The first parameter is the name of the index."

Reply:
+ same for the rest of the doc

Reply:
May be a list of Parameter / Type would be clearer